packages/blocks/github-pr-overview/src/overview.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from "react";
+import Grid from "@mui/material/Grid";
+import Avatar from "@mui/material/Avatar";
+import Stack from "@mui/material/Stack";
+import IconButton from "@mui/material/IconButton";
+import Tooltip from "@mui/material/Tooltip";
+import Cancel from "@mui/icons-material/Cancel";
+import { uniqBy } from "lodash";
+import moment from "moment";
+
+import {
+  GithubIssueEvent,
+  GithubPullRequest,
+  GithubReview,
+  isDefined,
+  PullRequestIdentifier,
+} from "./types";
+import { GithubPrTimeline } from "./timeline";
+
+export const Reviewer = (login: string, avatar_url?: string | null) => (
+  <Stack direction="row" spacing={1}>
+    <Avatar
+      alt={login}
+      src={avatar_url ?? undefined}
+      sx={{ width: "0.8em", height: "0.8em" }}
+      style={{ alignSelf: "center" }}
+    />
+    <span>{login}</span>
+  </Stack>
+);
+
+export type GithubPrOverviewProps = {
+  pullRequest: GithubPullRequest;
+  reviews: GithubReview[];
+  events: GithubIssueEvent[];
+  setSelectedPullRequestId: (x?: PullRequestIdentifier) => void;
+};
+
+export const GithubPrOverview: React.FunctionComponent<
+  GithubPrOverviewProps
+> = ({ pullRequest, reviews, events, setSelectedPullRequestId }) => {
+  const uniqueReviewers = uniqBy(
+    reviews.map((review) => {
+      return { login: review.user!.login, avatar_url: review.user!.avatar_url };
+    }),
+    "login",
+  );
+
+  const timeToClose =
+    pullRequest.closed_at != null
+      ? moment(pullRequest.closed_at).from(moment(pullRequest.created_at), true)
```

Comment:
`moment` is a very heavy package (200+KB minified) for date formatting. I'd use [`date-fns` ](https://date-fns.org/)

Reply:
Have swapped, cheers